exercises/practice/ledger/.meta/example.cairo

```diff
@@ -0,0 +1,225 @@
+// Refactored code
+use core::to_byte_array::{AppendFormattedToByteArray};
```

Comment:
Don't forget to [add a comment](https://github.com/exercism/cairo/issues/196#issuecomment-2516411071) on which changes you made when you refactored the solution

Reply:
It doesn't have to go into a lot of detail, just a short list summarizing the changes you made